bench/bench._test.go

```diff
@@ -0,0 +1,69 @@
+package bench
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/rand"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/tetratelabs/wazero/wasi"
+	"github.com/tetratelabs/wazero/wasm"
+	"github.com/tetratelabs/wazero/wasm/naivevm"
+	"github.com/tetratelabs/wazero/wasm/wazeroir"
+)
+
+func BenchmarkEngines(b *testing.B) {
+	buf, _ := os.ReadFile("case/case.wasm")
+	mod, _ := wasm.DecodeModule((buf))
+	b.Run("naivevm", func(b *testing.B) {
+		store := newStore(naivevm.NewEngine())
+		_ = store.Instantiate(mod, "test")
+		runBase64Benches(b, store)
+		runFibBenches(b, store)
+	})
+	b.Run("wazeroir_interpreter", func(b *testing.B) {
```

Comment:
maybe leave out _interpreter until there's an option? that would match `wazeroir.NewEngine()` more cleanly and likely lead to aligned tabs while this coexists with naievevm

Reply:
agreed!